tests/zkevm/test_worst_compute.py

```diff
@@ -955,3 +957,98 @@ def test_empty_block(
         post={},
         blocks=[Block(txs=[])],
     )
+
+
+@pytest.mark.valid_from("Cancun")
+def test_worst_bn128_pairings(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+):
+    """Test running a block with as many BN128 pairings as possible."""
+    env = Environment()
+
+    base_cost = 45_000
+    pairing_cost = 34_000
```

Comment:
LMK if these costs can be pulled from somewhere. If not, we can leave it this way.